src/ui/button/button.jsx

```diff
@@ -137,16 +142,23 @@ export function AuthButton(props : ButtonProps) : ElementNode {
                 onClick={ clickHandler }
                 onKeyPress={ keypressHandler }
                 tabindex='0'>
-
-                <div class={ CLASS.BUTTON_LABEL }>
-                    <PPSymbol style={ style } />
-                    <Logo style={ style } />
-                    {
-                        fundingSource === FUNDING.CREDIT ?
-                            <PPCreditLogo style={ style } locale={ locale } /> :
-                            <Label style={ style } content={ content } />
-                    }
-                </div>
+                    
+                { displayLabel &&
+                    <div class={ CLASS.BUTTON_LABEL }>
+                        <PPSymbol style={ style } />
+                        {
+                            fundingSource === FUNDING.CREDIT ?
+                                <PPCreditLogo style={ style } locale={ locale } /> :
+                                <Label style={ style } content={ customLabel !== undefined ? customLabel : 'Connect with PayPal' } />
+                        }
+                    </div>}
+                {
+                    !displayLabel &&
+                    <div className="PAYPAL-SPINNER"><SpinnerPage
```

Comment:
can we have the spinner as the `prerenderedTemplate` prop, here : https://github.com/paypal/paypal-identity-components/blob/master/src/zoid/button/prerender.jsx#L49 ?